scripts/EN/Harry

```diff
@@ -4472,9 +4472,9 @@ LoadBackground(0x7b); // 0x5836: 00001069
 LoadEffect(0xe, 0x1, 0x0); // 0x583c: 0000306d
 DrawBackground(); // 0x5846: 00000065
 PlayMusic(0x68); // 0x584a: 0000106c
-ShowText((1459)"The town has turned into that nightmarish world. It's as if the danger in this town increases with every second. In order to find Lisa I turn on the flashlight on my chest and begin to run at full speed."); // 0x5850: 0000100c
+ShowText((1459)"The town has turned into that nightmarish world.¥nW2It's as if the danger in this town increases with every second.¥nW2In order to find Lisa W1I turn on the flashlight on my chest and begin to run at full speed."); // 0x5850: 0000100c
 HandleInput(); // 0x5856: 00000010
-ShowText((1460)"I left the antique shop with the intention of reuniting with Lisa, but I discover that all the roads leading to the hospital are blocked. With no other choice I enter the shopping mall ¥"Town Center¥" in the shopping district."); // 0x585a: 0000100c
+ShowText((1460)"I left the antique shop with the intention of reuniting with Lisa, W1but I discover that all the roads leading to the hospital are blocked.¥nW2With no other choice I enter W1the downtown's premier shopping mall, W1¥"Silent Hill Town Center.¥""); // 0x585a: 0000100c
```

Comment:
Added Silent Hill to Town Center.

I also re-evaluated the last sentence and made a fairly large change. "Shopping district" becomes "downtown." Quite literally this is "busy section of the city" and while shopping district was not exactly wrong, downtown seems more suitable.

Massive cheers to Ilducci for pointing out that when collecting the Central Silent Hill map it actually does say "Shopping District" in the PS1 game!

"Central Silent Hill" could almost be used, but I don't believe that is the intent behind 繁華街.

I also restored the "premier" part. Literally the Japanese calls the Town Center the "best or top shopping mall of the downtown," but since there is really only one mall visible I felt "premier" was the best choice in English.